site/ru_RU.all.json

```diff
@@ -1023,10 +1023,12 @@
     "other": "Цена"
   },
   "shopping_price_plan_note": {
-    "zero": "{{ .Interval }} каждые .интервал.",
     "one": "{{ .Interval }} каждые после окончания 24-часового бесплатного пробного периода.",
```

Comment:
Russian requires [one, few, many, and other](https://github.com/nicksnyder/go-i18n/blob/0dc1626d56435e9d605a29875701721c54bc9bbd/i18n/language/codegen/plurals.xml#L176).